src/main/java/com/nu/art/http/HttpModule.java

```diff
@@ -187,6 +188,18 @@ public BaseTransaction() {
 		}
 	}
 
+	private HashMap<String, Getter<String>> defaultHeaders;
+
+	public void addDefaultHeader(String key, Getter<String> value) {
+		if (defaultHeaders == null)
```

Comment:
No need for this condition.. create the instance at the declaration

Reply:
Added map creation at declaration, and removed the null check.